src/server/scripts/scraper.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+//destructing the command line arguments
+const [,,...args]=process.argv;
```

Comment:
Please add a space between and after the `=`. And we are not using `;`, please remove it here and in the other lines.